tbd-fe/src/containers/App/App.scss

```diff
@@ -18,3 +18,9 @@ h1 {
     margin: 20px auto;
     width: 30px;
 }
+
+.result-background {
```

Comment:
Are moving away from the light theme that @jacquelinebelle was wireframing?

Reply:
It looked extremely bare. I can lower the opacity if needed. This is what it currently looks like...

![Screen Shot 2019-09-09 at 4 34 46 PM](https://user-images.githubusercontent.com/36940278/64571084-ea4ef200-d31f-11e9-9622-d673f85642ed.png)
![Screen Shot 2019-09-09 at 4 35 16 PM](https://user-images.githubusercontent.com/36940278/64571092-efac3c80-d31f-11e9-9644-24afddb318cb.png)

Reply:
Looks great!!